### PR TITLE
[cluster-test] Change num_nodes_down to percent_nodes_down

### PR DESCRIPTION
**Pull request by ankushagarwal**
## Summary

We were using num_nodes_down for `PerformanceBenchmarkNodesDown` experiment. This changes it to `percent_nodes_down`

### CODE REVIEW

File: testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs [hunk 1]
```diff
@@ -66,21 +66,25 @@ impl ExperimentParam for PerformanceBenchmarkNodesDownParams {
     type E = PerformanceBenchmarkNodesDown;
     fn build(self, cluster: &Cluster) -> Self::E {
         if self.is_fullnode {
-            let (down_instances, up_instances) =
-                cluster.split_n_fullnodes_random(self.num_nodes_down);
+            let num_nodes = cluster.fullnode_instances().len();
+            let nodes_down =
+                ((num_nodes as f64) * ((self.percent_nodes_down as f64) / 100f64)) as usize;
```
Comment (zekun000) [inline testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs]: is it better than `(num_nodes * self.percent_nodes_down) / 100`?

Comment (ankushagarwal) [reply testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs]: Updated

**Pull request closed without merging**